static/js/external/Makefile

```diff
@@ -0,0 +1,61 @@
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015 OpenCraft <xavier@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+# Variables ###################################################################
+
+GET = wget -qO
+CDNJS = https://cdnjs.cloudflare.com/ajax/libs
+
+
+# Files #######################################################################
+
+all: angular.min.js angular.min.js.map angular-mocks.js angular-route.min.js angular-route.min.js.map \
+	 angular-ui-router.min.js jquery.min.js jshint.js restangular.min.js underscore-min.js \
+	 underscore-min.map
+
+angular.min.js:
+	$(GET) $@ $(CDNJS)/angular.js/1.4.6/angular.min.js
```

Comment:
@smarnach It will reload them - the files are removed as part of the `make clean` rule, which is invoked before any execution. So it should always be up to date:

https://github.com/open-craft/opencraft/pull/5/files#diff-b67911656ef5d18c4ae36cb6741b7965R43